activesupport/test/core_ext/duration_test.rb

```diff
@@ -661,6 +661,22 @@ def test_durations_survive_yaml_serialization
     assert_equal 660, (d1 + 60).to_i
   end
 
+  def test_string_build_raises_error
+    error = assert_raises(TypeError) do
+      ActiveSupport::Duration.build("9")
+    end
+
+    assert_equal error.message, "can't build an ActiveSupport::Duration from a String"
```

Comment:
These assertions are backwards; the expected value should be passed first. See https://github.com/rails/rails/commit/a46b2f8911c5730c8c56d487b65f7c5627d334ee.

Reply:
@eugeneius: changed in https://github.com/rails/rails/pull/37013/commits/94fa3e86285c1f05a8f3bbaa20cbdad0e649323a

Reply:
@eugeneius: Squashed into https://github.com/rails/rails/pull/37013/commits/b8b7e853832ff65509c1c29d0fbe259b7ab5aded